src/gui/wizard/owncloudwizardcommon.cpp

```diff
@@ -68,6 +72,39 @@ namespace WizardCommon {
         errorLabel->setVisible(false);
     }
 
+    void customizeRadioButtonStyle(QRadioButton *radioButton)
+    {
+        auto radioButtonPalette = radioButton->palette();
+        radioButtonPalette.setColor(QPalette::Text, Qt::black);
+        radioButtonPalette.setColor(QPalette::Base, Qt::white);
+        radioButton->setPalette(radioButtonPalette);
+    }
+
+    void customizeSpinBoxStyle(QSpinBox *spinBox)
+    {
+        auto spinBoxPalette = spinBox->palette();
+        spinBoxPalette.setColor(QPalette::Base, Qt::white);
+        spinBoxPalette.setColor(QPalette::WindowText, Qt::black);
+        spinBoxPalette.setColor(QPalette::Text, Qt::black);
+        spinBox->setPalette(spinBoxPalette);
+    }
+
+    void customizeCheckBoxStyle(QCheckBox *checkBox)
+    {
+        auto checkBoxPalette = checkBox->palette();
+        checkBoxPalette.setColor(QPalette::Text, Qt::black);
+        checkBoxPalette.setColor(QPalette::Base, Qt::white);
+        checkBox->setPalette(checkBoxPalette);
+    }
+
+    void customizeSecondaryButtonStyle(QAbstractButton *button)
```

Comment:
I'll contradict myself, there would actually be a way to do that in a different way. But let's not get there yet. :-)